fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexingBase.java

```diff
@@ -401,35 +400,32 @@ public void enforceStampOverwrite() {
     @SuppressWarnings("PMD.CloseResource")
     private CompletableFuture<Void> setIndexingTypeOrThrow(FDBRecordStore store, boolean continuedBuild) {
         // continuedBuild is set if this session isn't a continuation of a previous indexing
-        Transaction transaction = store.getContext().ensureActive();
         IndexBuildProto.IndexBuildIndexingStamp indexingTypeStamp = getIndexingTypeStamp(store);
 
-        return forEachTargetIndex(index -> setIndexingTypeOrThrow(store, continuedBuild, transaction, index, indexingTypeStamp));
```

Comment:
Is there a way to change all the indexes' typestamp with a single transaction? (to avoid changing few indexes but not all..)

Reply:
Yes, the transactionality of the operation here hasn't changed. The only thing that changed was that the `Transaction` object is now no longer passed separately from the store, which I made because that way, it is clearer that the store and transaction are linked (i.e., that someone wasn't doing something sneaky like passing a store that was created with a _different_ transaction)